tests/e2e/network/service_annotations.go

```diff
@@ -906,6 +906,30 @@ func getAzureLoadBalancerFromPIP(tc *utils.AzureTestClient, pip, pipResourceGrou
 	return &lb
 }
 
+func getAzureInternalLoadBalancerFromPrivateIP(tc *utils.AzureTestClient, ip, lbResourceGroup string) *network.LoadBalancer {
```

Comment:
Why this function is defined here? It is used in `ensureloadbalancer.go` instead of `service_annotation.go`. I suggest putting it in `ensureloadbalancer.go` or `network_utils.go` (preferred).

Reply:
I put it here because right above this function, there is `getAzureLoadBalancerFromPublicIP`. I'll put it in `ensureloadbalancer.go`, `network.utils.go` looks in a lower level (e.g. no `EXPECT()` etc)